beta-src/src/components/ui/WDNotificationContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { Stack } from "@mui/material";
+import { useAppSelector } from "../../state/hooks";
+import { gameNotifications } from "../../state/game/game-api-slice";
+import WDDeletePanel from "./WDDeletePanel";
+
+const WDNotificationContainer: React.FC = function (): React.ReactElement {
+  const notifications = useAppSelector(gameNotifications);
+  return (
+    <Stack direction="column">
+      {/* Notifications[0] will have a value if there player is in build phase and needs to destroy units */}
+      {notifications[0] ? (
+        <WDDeletePanel notifications={notifications} />
+      ) : null}
```

Comment:
How would this work with other phases? Ones where we only have a single notification or potentially more? Seems to me like this would only work during the "Builds" phase, particularly where destroy is used, but wouldn't translate well into the diplomacy phase.

Reply:
I'm currently thinking that each phase has a set of indexes reserved to them, so let's say build phase build units has indexes 2 and 3, we could add 
      {notifications[2] ? (
        <WDBuildPanel notifications={notifications} />
      ) : null}
underneath. Delete itself can have 1 or 2 notifications so for example you can just build WDBuildPanel, call WDNotification for each reserved index, and WDNotification renders conditionally, which is the case for Delete. I know indexes aren't the most descriptive but I wasn't sure on how to refactor notifications to be able to take a descriptive identifier that would improve readability without hitting performance. If you have any suggestions I am open to any.

Reply:
Actually, destructuring the array in WDNotificationContainer and WDDeletePanel does make it a lot more readable